src/analysis/normed_space/normed_group_hom.lean

```diff
@@ -389,7 +389,7 @@ def comp_hom : (normed_group_hom V₂ V₃) →+ (normed_group_hom V₁ V₂) 
 add_monoid_hom.mk' (λ g, add_monoid_hom.mk' (λ f, g.comp f)
   (by { intros, ext, exact g.map_add _ _ }))
   (by { intros, ext, simp only [comp_apply, pi.add_apply, function.comp_app,
-                                add_monoid_hom.add_apply, add_monoid_hom.coe_mk', coe_add] })
+                                add_monoid_hom.add_apply, add_monoid_hom.mk'_apply, coe_add] })
```

Comment:
What is the new statement of `mk'_apply`?

Reply:
It is the same as the current `add_monoid_hom.coe_mk'`, i.e. an equality between functions. I tried to keep the generated lemmas as close to the existing ones, except for the name.

It is currently not possible in the `@[simps]` framework to sometimes use a projection with one name `_apply` and sometimes with a different name `_coe` (though this would be easy to add).

Reply:
`_apply` feels like the wrong name here - I'm not sure we should be using `simps` to save lines if it comes at the expense of generating an unusual name.